platform-sdk/docs/proposals/TSS-Library/TSS-Library.md

```diff
@@ -52,6 +52,11 @@ the distribution aspect, the loading, and the in-memory interpretation from each
 - **Share**: Represents a piece of the necessary public/private elements to create signatures. In TSS,
   a threshold number of shares is needed to produce an aggregate signature that the ledger public key can later verify.
 - **Polynomial Commitment**: It enables verification of evaluations of the polynomial at specific points without revealing the entire polynomial.
+- **Participant**: Any party involved in the distributed key generation protocol. Their goal is to generate a public key and provide a set of receivers with matching secret shares of the secret key. 
```

Comment:
> Their goal is to generate a public key and provide a set of receivers with matching secret shares of the secret key.

This description is confusing to me. Is the public key you are referring to the ledger ID? What is "matching" about the secret shares?

IMO it's not necessary to state a goal of the participant here, since the goal is just to do the protocol described in this doc.

Reply:
I removed the goal part. 
In my head, the explanation made more sense than what ended up in the paper. I was thinking that the secret should be split into matching pieces for each share and encrypted with the matching participant's public key.